tests/bsim/bluetooth/host/adv/extended/tests_scripts/ext_adv_conn.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Copyright (c) 2024 Croxel, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Extended advertising test:
+#
+# - Connectable: In addition to broadcasting advertisements, it is connectable
```

Comment:
This test is nice, but I think we'll need an additional robustness test:
We should create separate builds for central & peripheral, both w/ only 1 available connection context.
The test would then do what you're doing here, but loop a few times and ensure we get reconnected every time. 
That's the main use-case of the automatic re-starting of the (legacy) advertiser. A lot of people have a small device that is configured w/ only 1 connection slot.

That said, I prefer this PR goes in like this and we add it in the next one.

Reply:
Sounds good.

Suggestion: Once this PR is merged, I can follow-up shortly-after with a PR where this additional more-isolated test is captured.